### PR TITLE
UI setup and tests for login and register use cases

### PR DESCRIPTION
**Pull request by modaserMoj**
Modified the login control to properly execute the get and post requests
Modified the signup control to properly execute the get and post requests
Created the login and sign up html files
Created the tests for the login and register use cases

### CODE REVIEW

**Review by youssefsoli — APPROVED** [review]
Everything looks up to check, all tests are passing and the UI looks better than ever, good job.

File: src/main/resources/templates/home.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (jpablo2002) [inline src/main/resources/templates/home.html:1]: I believe this document can be removed, as the feed.html file serves this purpose.

**Review by jpablo2002 — CHANGES_REQUESTED** [review]

**Review by youssefsoli — APPROVED** [review]
Everything looks up to check, all tests are passing and the UI looks better than ever, good job.

**Review by jpablo2002 — APPROVED** [review]
Great implementation of the login/register use cases following clean architecture! Also good testing covering various different scenarios, as well as well-documented to make the code easy to follow!

**Review by feiwun — APPROVED** [review]
UI setup and tests pass the tests and provide intended functions

**Pull request merged**